app/assets/stylesheets/application.scss

```diff
@@ -49,3 +49,14 @@
     display:none;
   }
 }
+
+#subnavbar {
+  font-size: 1.5em;
+  margin-top: 20px;
+  text-align: center;
+}
+
+#subnavbar a {
+  text-decoration: none;
+  color: #2F6A9A;
+}
```

Comment:
Try not to put this code in `application.scss` we want to keep that as a manifest file as much as possible.

Also you can take advantage of SASS by writing this as:
```css
#subnavbar {
   font-size: 1.5em;
   margin-top: 20px;
   text-align: center;
  a {
    text-decoration: none;
    color: #2F6A9A;
  }
}
```